source/Cosmos.System2/Kernel.cs

```diff
@@ -1,204 +1,178 @@
 using System;
+using System.Drawing;
 using sysIO = System.IO;
 using Cosmos.Debug.Kernel;
 using Cosmos.HAL;
 using Cosmos.System.ScanMaps;
 
-namespace Cosmos.System
+namespace Cosmos.System;
```

Comment:
Are these changes just whitespace changes? I can't tell because of all the formatting changes.

Reply:
yes Rider reformatted the file